src/frontend/handlers/__snapshots__/invoice.test.ts.snap

```diff
@@ -279,6 +281,9 @@ exports[`invoice handler Page displays Invoice above threshold banner text and t
       </tbody>
     
   </table>
+  <p class="govuk-body-s govuk-!-padding-bottom-3">
+    <span class="govuk-!-font-weight-bold">Note:</span> If there is no invoiced quantity because invoice data is missing, "Invoice data missing" will be displayed, If there is no measured quantity because events are missing, display "Events missing" will be displayed. In either situation occurs, the quantity discrepancy column will be blank.
```

Comment:
```suggestion
    <span class="govuk-!-font-weight-bold">Note:</span> If there is no invoiced quantity because invoice data is missing, "Invoice data missing" will be displayed, If there is no measured quantity because events are missing, "Events missing" will be displayed. In either situation occurs, the quantity discrepancy column will be blank.
```

Reply:
(I think the message in the ticket has an unnecessary word.)